apps/omg_watcher/config/dev.exs

```diff
@@ -41,13 +41,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :omg_watcher, OMG.Watcher.DB.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "omisego_dev",
-  password: "omisego_dev",
-  database: "omisego_dev",
-  hostname: "localhost",
-  pool_size: 10,
-  timeout: 60_000,
-  connect_timeout: 60_000
+  pool: Ecto.Adapters.SQL.Sandbox,
+  url: {:system, "DATABASE_URL", "postgres://omisego_dev:omisego_dev@localhost/omisego_dev"}
```

Comment:
Pls find web resource and conform to the naming

Reply:
I added the following comment `# DATABASE_URL format is following `postgres://{user_name}:{password}@localhost/{database_name}``